### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by michaelestorgio**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by juanestorgio** [comment]
Hi

**Comment by michaelestorgio** [comment]
Nice

**Review by michaelestorgio — COMMENTED** [review]
Fantabulous

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (michaelestorgio) [inline index.html:47]:
```suggestion
  </tr>
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (michaelestorgio) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again my friend 🧑‍💻 🤖!</h2>
```

**Pull request merged**